cocotb/triggers.py

```diff
@@ -791,6 +793,8 @@ class First(_AggregateWaitable):
 
     Returns the result of the trigger that fired.
 
+    This is cocotb's equivalent to Verilog's ``join_any``.
```

Comment:
I might go for the weaker wording of "This is similar to Verilog's ``join_any``"

Reply:
Changed.